Code/Tools/AzTestRunner/src/main.cpp

```diff
@@ -222,6 +229,10 @@ namespace AzTestRunner
         if (testMainFunction->IsValid())
         {
             result = (*testMainFunction)(argc, argv);
+            if (result != 0)
+            {
+                std::cout << retry_command << std::endl;
+            }
             std::cout << "OKAY " << symbol << "() returned " << result << std::endl;
```

Comment:
Retry command should be output after stating which test returned what. Should probably build it here in the failure block. While doing so would save s a minuscule amount of perf during test passes, it helps future maintenance to keep the variable defined near where it is used.

Reply:
I tried building it inside of the failure block, but I think the args are getting de-referenced sometime before that because I was unable to do so. I can move the retry command after the return code though.

Reply:
moved construction and displays after the test ends